localization/yabloc/README.md

```diff
@@ -0,0 +1,27 @@
+# YabLoc
+
+**YabLoc** is vision-baed localization with vector map. [https://youtu.be/Eaf6r_BNFfk](https://youtu.be/Eaf6r_BNFfk)
+
+[![thumbnail](docs/yabloc_thumbnail.jpg)](https://youtu.be/Eaf6r_BNFfk)
+
```

Comment:
Would you also add an architecture overview of YabLoc?
I think the connection between YabLoc and other Autoware topics is also helpful

Reply:
Nice idea! I added a diagram in eb70a60